components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/Fields.java

```diff
@@ -33,6 +33,7 @@ public enum Fields {
     EXECUTED_NOT_CACHEABLE_DURATION("Executed not cacheable duration", d -> totalDuration(d, "executed_not_cacheable")),
     EFFECTIVE_TASK_EXECUTION_DURATION("Effective task execution duration", d -> String.valueOf(d.getEffectiveTaskExecutionDuration().toMillis())),
     SERIALIZATION_FACTOR("Serialization factor", d -> formatBigDecimal(d.getSerializationFactor())),
+    EXECUTED_CACHEABLE_DURATION_MILLISECONDS("Executed cacheable duration milliseconds", d -> totalDurationMillis(d, "executed_cacheable")),
```

Comment:
I added this as an extra field so that these changes are backwards compatible with the current version of the `build-scan-support-tool`.

I plan to create a follow up PR which removes all special logic from the build validation scripts to use once the `build-scan-support-tool` is updated.

Reply:
This has been cleaned up here: https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/283